fullstack-templates/saas/package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next build && cargo shuttle run",
+    "dev": "ROARR_LOG=true turbowatch ./turbowatch.ts | roarr",
```

Comment:
Will this work on Windows?

Reply:
Regarding Windows support, I am unsure, I have now added cross-env so that it should theoretically work, but since I don't have a Windows machine available, I can't test it at the moment

Reply:
> Let's hope it does. It's not the end of the world if it does not.

I made a few changes to the turbowatch script and had a friend test it under Windows, it should now work. He also checks whether the shuttle-runtime is installed or not